modules/apps/notification/notification-service/src/main/java/com/liferay/notification/internal/upgrade/v2_2_0/NotificationQueueEntryUpgradeProcess.java

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.notification.internal.upgrade.v2_2_0;
+
+import com.liferay.portal.kernel.upgrade.UpgradeProcess;
+import com.liferay.portal.kernel.upgrade.UpgradeProcessFactory;
+import com.liferay.portal.kernel.upgrade.UpgradeStep;
+
+/**
+ * @author Feliphe Marinho
+ */
+public class NotificationQueueEntryUpgradeProcess extends UpgradeProcess {
+
+	@Override
+	protected void doUpgrade() throws Exception {
+		runSQL("UPDATE NotificationQueueEntry SET type_ = 'email'");
```

Comment:
@MarinhoFeliphe Is it possible to use PreparedStatement  and Connection here?

Like in [this upgrade process](https://github.com/liferay/liferay-portal/blob/master/modules/apps/object/object-service/src/main/java/com/liferay/object/internal/upgrade/v3_24_0/ObjectFieldSettingUpgradeProcess.java)

Reply:
Hi @gabrielwas 

I think so, but Brian asked for me to replace to runSQL, I'm trying to find his comment requesting this change.

Reply:
Hm... maybe it's because the update syntax doesn't change according to the dbms connector :thinking: 

I asked this because we've faced some issues due to the variety of databases syntax and we've had to adapt it to use SQLTransformer and Connection.

Nevermind then, we can forward it as it is.